src/Transformer/Normalization/DefaultTransformer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flagbit\Bundle\CategoryBundle\Transformer\Normalization;
+
+use Flagbit\Bundle\CategoryBundle\Transformer\NormalizationTransformer;
+
+/**
+ * Default normalization transformer that applies no changes.
+ */
+class DefaultTransformer implements NormalizationTransformer
+{
+    /**
+     * @phpstan-param mixed $data
+     *
+     * @return mixed
+     */
+    public function transform($data)
+    {
+        return $data;
```

Comment:
It usually should get a string with `$data`, so I think that's correct for the DefaultTransformer.